backend/src/main/resources/db/migration/V002__create-transaction-categories.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE TRANSACTION_CATEGORIES (
+	id INT GENERATED ALWAYS AS IDENTITY,
+	name varchar(255) NOT NULL,
+	description varchar(255),
+	primary key (id)
+);
+
+INSERT INTO TRANSACTION_CATEGORIES (name, description) VALUES ('Salário', 'Salário mensal');
+INSERT INTO TRANSACTION_CATEGORIES (name, description) VALUES ('Contas', 'Contas da casa');
```

Comment:
Dúvida: esses inserts, caso já tenham sido criados no banco, quando a aplicação for redeployada ela irá adicionar esses valores repetidos ?

Reply:
Outro ponto: o que acha de deixar o campo name desses inserts de forma padronizada, lowercase e tals? 

aí no front a gente usa a descrição para exibir na tela

Reply:
O flyway cria uma tabela no banco onde tem os scripts que já rodaram, no caso do deploy como esse script já foi rodado não vai tentar criar em cima, fiz esse script com base no estado atual do banco.

Entendi agora o que você tinha falado para formatar a categoria, vou fazer.

Reply:
Isso, comentei dessa padronização aqui também  porque senão tivermos um padrão, podemos inserir coisas iguais escritas de jeitos diferentes, tipo "Salário" e "salario".